api_types.ts

```diff
@@ -351,40 +351,168 @@ export interface Network {
+ * An HTTP request used with the {@link Fetcher}.
+ *
+ * The structure largely follows https://developer.mozilla.org/en-US/docs/Web/API/Request
+ */
 export interface FetchRequest {
+  /** The HTTP method/verb (e.g. GET or POST). */
   method: FetchMethodType;
+  /**
+   * The URL to connect to. This is typically an absolute URL, but if your
+   * pack uses authentication and {@link requiresEndpointUrl} and so has a unique
+   * endpoint per user account, you may also use a relative URL and Coda will
+   * apply the user's endpoint automatically.
+   */
   url: string;
+  /**
+   * The body of the HTTP request, if any.
+   *
+   * If you are sending a JSON payload, make sure to call `JSON.stringify()` on the object payload.
+   */
   body?: string;
+  /**
+   * Key-value form fields, if submitting to an endpoint expecting a URL-encoded form payload.
+   */
   form?: {[key: string]: string};
+  /**
+   * HTTP headers. You should NOT include authentication headers, as Coda will add them for you.
+   */
   headers?: {[header: string]: string};
-  // Allows explicit caching of the results of this request.
+  /**
+   * A time in seconds that Coda should cache the result of this HTTP request.
+   *
+   * Any time that this pack makes the same FetchRequest, a cached value can be returned
+   * instead of making the HTTP request. If left unspecified, Coda will automatically
+   * cache all GET requests for approximately 5 minutes. To disable the default caching,
+   * set this value to `0`.
+   */
   cacheTtlSecs?: number;
-  // Allows binary responses.
+  /**
+   * Indicates that you expect the response to be binary data, instructing Coda
+   * not to attempt to parse the response in any way. Otherwise, Coda may attempt
+   * to parse the response as a JSON object.
+   */
   isBinaryResponse?: boolean;
-  // Disable authentication
+  /**
+   * If true, Coda will not apply authentication credentials even if this pack is
+   * configured to use authentication. This is very rare, but sometimes you may
+   * wish to make an unauthenticated supporting request as part of a formula implementation.
+   */
   disableAuthentication?: boolean;
 }
 
-// Copied from https://developer.mozilla.org/en-US/docs/Web/API/Response
+/**
+ * The response of a call to {@link Fetcher.fetch}.
+ *
+ * The structure largely follows https://developer.mozilla.org/en-US/docs/Web/API/Response
+ */
 export interface FetchResponse<T extends any = any> {
+  /** The HTTP status code, e.g. `200`. */
   status: number;
+  /**
+   * The body of the response.
+   *
+   * If the response contains JSON data, either because the Content-Type header is application/json
+   * or if the data is JSON-parseable, this will be a parsed JavaScript object.
+   * Similarly, if the response headers are text/xml or application/xml, this will be a parsed
+   * JavaScript object using the `xml2js` library.
+   *
+   * If implicit parsing is undesirable, you may use {@link isBinaryResponse} on the request
```

Comment:
Should you mention that it returns a Node.js Buffer in this case?

Reply:
Yep good point, updated.